src/ci/resources/aws_application.conf

```diff
@@ -33,16 +33,17 @@ backend {
         numSubmitAttempts = 3
         numCreateDefinitionAttempts = 3
         // Base bucket for workflow executions
-        root = "s3://cromwell-centaur-execution/cromwell-execution"
+        root = "s3://cromwell-centaur-execution-new/cromwell-execution"
 
         // A reference to an auth defined in the `aws` stanza at the top.  This auth is used to create
         // Jobs and manipulate auth JSONs.
         auth = "default"
 
-        concurrent-job-limit = 16
+        concurrent-job-limit = 24
```

Comment:
Yeah, I was futzing with it when the ci/cd test was timing out but that was a different issue. I might leave it the same if it doesn't break stuff as it could speed up the test phase